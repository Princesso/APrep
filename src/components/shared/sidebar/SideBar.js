import React from 'react';
import styled from 'styled-components'
import colors from '../../../styles/colors'

const StyledSideBar= styled.div`
  background: ${colors.gray};
  height: 50px;
 
`
const SideBar = () => {
  return(
    <StyledSideBar>
      <div className="header"> sidebar</div>
    </StyledSideBar>
  )
}

export default SideBar