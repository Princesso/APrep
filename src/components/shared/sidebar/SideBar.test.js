import React from 'react';
import SideBar from './SideBar';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import Store from '../../../store';
import { BrowserRouter, Route, Switch } from 'react-router-dom'

test('Side Bar', () => {
  const component = renderer.create(
    <Provider store={Store}>
      <BrowserRouter>
          <Switch>
            <SideBar/>
          </Switch>
      </BrowserRouter>
    </Provider>
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})