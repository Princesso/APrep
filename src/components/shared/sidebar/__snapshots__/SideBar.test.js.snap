// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side Bar 1`] = `
<div
  className="sc-bdVaJa fVNxTI"
>
  <div
    className="header"
  >
     sidebar
  </div>
</div>
`;
